support/ppPrint.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
```

Comment:
`#pragma once`is not standard and should generally be avoided. We tend to use non-standard compilers sometimes which means it's more likely to be a pain point for us than average.

Reply:
Ok thanks, I will stop using it from now on.